tests/components/homekit/test_type_switches.py

```diff
@@ -2,12 +2,52 @@
 import pytest
 
 from homeassistant.core import split_entity_id
-from homeassistant.components.homekit.type_switches import Switch
+from homeassistant.components.homekit.type_switches import Outlet, Switch
 from homeassistant.const import ATTR_ENTITY_ID, STATE_OFF, STATE_ON
 
 from tests.common import async_mock_service
 
 
+async def test_outlet_set_state(hass, hk_driver):
+    """Test if Outlet accessory and HA are updated accordingly."""
+    entity_id = 'switch.outlet_test'
+
+    hass.states.async_set(entity_id, None)
+    await hass.async_block_till_done()
+    acc = Outlet(hass, hk_driver, 'Outlet', entity_id, 2, None)
+    await hass.async_add_job(acc.run)
```

Comment:
You need to add `await hass.async_block_till_done()` afterwards.

Reply:
If you're proposing to add `await hass.async_block_till_done()` after `await hass.async_add_job(acc.run)`, shouldn't the same be done for all other test types? Why is `test_outlet_set_state` different?

Reply:
It isn't different. I already added the missing ones in #14689